source/accessibility.html.md.erb

```diff
@@ -0,0 +1,120 @@
+---
+title: Accessibility statement for GOV.UK Pay's technical documentation
+last_reviewed_on: 2020-09-01
+review_in: 6 months
+hide_in_navigation: true
+---
+
+# Accessibility statement for GOV.UK Pay technical documentation
+
+This accessibility statement applies to the GOV.UK Pay technical documentation at [https://docs.payments.service.gov.uk/](https://docs.payments.service.gov.uk/).
+
+This website is run by the GOV.UK Pay team at the Government Digital Service (GDS). We want as many people as possible to be able to use this website. For example, that means you should be able to:
+
++ change colours, contrast levels and fonts
++ zoom in up to 300% without the text spilling off the screen
++ navigate most of the website using just a keyboard
++ navigate most of the website using speech recognition software
++ listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)
+
+We’ve also made the website text as simple as possible to understand.
+
+[AbilityNet](https://mcmw.abilitynet.org.uk/) has advice on making your device easier to use if you have a disability.
+
+### How accessible this website is
+
+We know some parts of this website are not fully accessible.
+
+The following pages have redundant links:
```

Comment:
what do we mean by 'redundant' here?

Reply:
Redundant means that sequential links are the same link. It often happens when you have multiple "Contact us" links on a page, with no other links in between them. That is the case here. I've checked with the accessibility community. It's not a fail, but it does need to be called out in the statement.